src/core/Akka/Util/Internal/Extensions.cs

```diff
@@ -72,10 +72,30 @@ public static T Head<T>(this IEnumerable<T> self)
         /// <returns>The path elements</returns>
         public static IEnumerable<string> SplitDottedPathHonouringQuotes(this string path)
         {
-            return path.Split('\"')
-                .AlternateSelectMany(
```

Comment:
Will move to tests since there is dependence in test-oracle scenario now. Just was hesitating to remove public method.